spec.html

```diff
@@ -26779,6 +26779,7 @@ <h1>Abstract Module Records</h1>
               <td>
                 <p>Returns a promise for the evaluation of this module and its dependencies, resolving on successful evaluation or if it has already been evaluated successfully, and rejecting for an evaluation error or if it has already been evaluated unsuccessfully. If the promise is rejected, hosts are expected to handle the promise rejection and rethrow the evaluation error.</p>
                 <p>Link must have completed successfully prior to invoking this method.</p>
+                <p>A non Cyclic Module Record must always returns a settled promise. This is asserted in InnerModuleEvaluation, and a non Cyclic Module Record is evaluated with EvaluateModuleSync.</p>
```

Comment:
```suggestion
                <p>A non Cyclic Module Record must always returns a settled promise.</p>
```

From a host point view the only thing that matters is what they have to do, and not what algorithms would break if they don't. Similarly, for "Link must have completed successfully prior to invoking this method" we do not say that that's asserted in step 2 of Evaluate.

(and if we do need to link, we should do it so through `<emu-xref>` so that it stays up to date)